src/OpenTelemetry.AutoInstrumentation/Plugins/PluginManager.cs

```diff
@@ -24,15 +24,15 @@ namespace OpenTelemetry.AutoInstrumentation.Plugins;
 
 internal class PluginManager
 {
-    private readonly IReadOnlyList<(Type Type, object Instance)> _plugins;
+    private readonly IReadOnlyList<(Type Type, object? Instance)> _plugins;
```

Comment:
`Instance` should not be nullable here. The dictionary must have an instance for the type.